appserver/tests/v2-tests/appserv-tests/devtests/connector/SunRaXml/cci-embedded/build.xml

```diff
@@ -131,7 +131,7 @@
                         <param name="tx.mode" value="tx"/>
                 </antcall>
 		-->
-                <antcall target="reconfig-common"/>
+                <!--antcall target="reconfig-common"/-->
```

Comment:
Please delete this call (here and in all occurrences below) instead of commenting